components/brave_wallet/common/brave_wallet_types.h

```diff
@@ -11,29 +11,25 @@
 #include <vector>
 
 #include "base/values.h"
-#include "boost/multiprecision/cpp_int.hpp"
 #include "third_party/abseil-cpp/absl/types/optional.h"
 
 namespace brave_wallet {
 
-typedef boost::multiprecision::uint256_t uint256_t;
-typedef boost::multiprecision::int256_t int256_t;
+typedef unsigned _BitInt(256) uint256_t;
+typedef _BitInt(256) int256_t;
 
-typedef boost::multiprecision::uint128_t uint128_t;
-typedef boost::multiprecision::int128_t int128_t;
-
-// Note that boost's int256/128_t has 256/128 precision bits and it uses an
-// extra sign bit so its max and min value differs from 2's complement types.
+typedef unsigned _BitInt(128) uint128_t;
+typedef _BitInt(128) int128_t;
 
 // 2^255 - 1
-constexpr int256_t kMax256BitInt = std::numeric_limits<int256_t>::max() >> 1;
+constexpr int256_t kMax256BitInt = std::numeric_limits<int256_t>::max();
 // -(2^255 -1)
-constexpr int256_t kMin256BitInt = std::numeric_limits<int256_t>::min() >> 1;
+constexpr int256_t kMin256BitInt = std::numeric_limits<int256_t>::min();
 
 // 2^128 - 1
-constexpr int128_t kMax128BitInt = std::numeric_limits<int128_t>::max() >> 1;
+constexpr int128_t kMax128BitInt = std::numeric_limits<int128_t>::max();
 // -(2^128 -1)
-constexpr int128_t kMin128BitInt = std::numeric_limits<int128_t>::min() >> 1;
+constexpr int128_t kMin128BitInt = std::numeric_limits<int128_t>::min();
```

Comment:
I believe some comments above are incorrect now :)
Actually do we need these constants? The only place references them is Base10ValueToInt256 which should be also be fixed now as it compares int256_t with it's true min/max.

Reply:
Those comments are correct, it was shifted not minus

Reply:
And of course we can use  std::numeric_limits<*> directly.

Reply:
Oh I see what you mean by incorrect now, that 128 bit comment was added by different person and it is indeed wrong.

Reply:
comments fixed and force pushed.